examples/DotNetCore/WebJobs/WebJobHelloWorld/WebJobHelloWorld/Functions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace WebJobHelloWorld
+{
+    public class Functions
+    {
+        public static void ProcessQueueMessage(
+            [QueueTrigger("%QueueName%")] string message, // Get queue name from config
+            ILogger logger)
+        {
```

Comment:
We discussed, if this is not from a Visual Studio template or something similar, we can add some comments here to explain this is where the user code of the web job goes and as an example, we just log the message.

Reply:
Added comments here.